actions/concurrent_runner.go

```diff
@@ -31,7 +31,7 @@ const defaultTimeout = time.Second * 10
 type ActionResult struct {
 	ID     string
 	Output []byte
-	Error  error
+	Error  string
```

Comment:
Why did you decide to use string instead of error?

Reply:
Mainly to simplify test – `signal: killed` error is an unexported type.
We converted it to `string` in the caller, so nothing changed in the end.